### PR TITLE
Added story for dnn-dropzone

### LINKED ISSUE #508: Implement story for `dnn-dropzone`
Implement story for `dnn-dropzone`

### PR DESCRIPTION
**Pull request by david-poindexter**
Resolves #508 

image

### CODE REVIEW

File: src/components/dnn-dropzone/dnn-dropzone.stories.ts [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+            ?allow-camera-mode=${ifDefined(args.allCameraMode)}
+            capture-quality=${ifDefined(args.captureQuality)}>
+        </dnn-dropzone>
+    `;
+
+export const Dropzone = Template.bind({});
+Dropzone.args = {
+    allowedExtensions: 'jpg,jpeg,png,gif',
+    allowCameraMode: false,
+    captureQuality: 0.8,
```
Comment (valadas) [inline src/components/dnn-dropzone/dnn-dropzone.stories.ts:57]:
```suggestion
    captureQuality: 0.8,
    resx,
```

Here the `const resx` was declared on line 29 but never used throwing the build error and also I would guess updating the value would have not affected the component in the story.

**Review by valadas — CHANGES_REQUESTED** [review]
One little thing to fix the build

**Comment by david-poindexter** [comment]
@valadas I merged your suggestion, so I think this one is good to merge now.

**Review by valadas — APPROVED** [review]
Thanks

**Pull request merged**